README.md

```diff
@@ -16,6 +16,8 @@ Launch NFD and check "NFD is started."
 
 npChat can communicate via NFD's Wifi Direct feature or over a network that supports UDP multicast.
```

Comment:
Isn't this line enough? Maybe it can be modified as:
npChat can communicate via:
1) NFD's Wifi Direct feature: Within the NFD app setup WiFi direct between phones manually. Usually this is done once and NFD remembers. If there are problems, please force close NFD and setup the WiFi direct again. More details about NFD's wifi direct feature can be found [here](https://redmine.named-data.net/projects/nfd-android/wiki/NDN_Over_WiFi_Direct_Protocol_Specification).
or
2) Automatically over a network that supports UDP multicast.

Reply:
Agreed, you want me to update this on readme?

Reply:
Yes. On point 2 add another line: Please note that UDP multicast has been known to exhibit higher packet loss than UDP unicast over WiFi direct.